lapack/native/general.go

```diff
@@ -57,12 +57,10 @@ const (
 	badVn2          = "lapack: vn2 has insufficient length"
 	badUplo         = "lapack: illegal triangle"
 	badWork         = "lapack: insufficient working memory"
-	badWorkStride   = "lapack: insufficient working array stride"
```

Comment:
This needs to be reflected in the netlib repository. Please make an issue if it doesn't already match.